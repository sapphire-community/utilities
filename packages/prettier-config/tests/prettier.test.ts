// This has to be imported with require!
// eslint-disable-next-line @typescript-eslint/no-var-requires
const prettierConfig = require('../src');

describe('Prettier Config', () => {
	test('should export rules', () => {
		expect(prettierConfig.$schema).toBe('http://json.schemastore.org/prettierrc');
		expect(prettierConfig.useTabs).toBe(true);
		expect(prettierConfig).toMatchSnapshot();
	});
});
