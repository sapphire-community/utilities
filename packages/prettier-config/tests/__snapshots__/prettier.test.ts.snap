// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prettier Config should export rules 1`] = `
Object {
  "$schema": "http://json.schemastore.org/prettierrc",
  "endOfLine": "lf",
  "overrides": Array [
    Object {
      "files": ".all-contributorsrc",
      "options": Object {
        "parser": "json",
      },
    },
    Object {
      "files": "*.yml",
      "options": Object {
        "tabWidth": 2,
        "useTabs": false,
      },
    },
  ],
  "printWidth": 150,
  "quoteProps": "as-needed",
  "semi": true,
  "singleQuote": true,
  "tabWidth": 4,
  "trailingComma": "none",
  "useTabs": true,
}
`;
